ui/lib/sync/addon/routes/secrets.ts

```diff
@@ -13,13 +13,8 @@ export default class SyncSecretsRoute extends Route {
   @service declare readonly router: RouterService;
   @service declare readonly flags: FlagService;
 
-  beforeModel() {
```

Comment:
Commented in slack about refresh ramifications removing this fetch to only happen in the cluster.js route